src/composables/LetoModelizerApi.js

```diff
@@ -57,3 +57,23 @@ export async function getUserInformation(sessionToken) {
 
   return axios.get('/backend/api/users/me', headers);
 }
+
+/**
+ * Get all roles of a user.
+ * @param {string} userId - User id.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object[]>} Return an array of roles. Role is a parseObject server, so see
+ * documentation on api to have more information.
+ */
+export async function getUserRoles(userId, sessionToken) {
```

Comment:
Did you check if there is a way to have the roles in the one same call ?
For instance using the /users/me endpoint and with adding some queryParam or something like that ?

Like this we would retrieve everything at once, instead of 2 separate calls

Reply:
yes and after all my tests this is the only call that give me all the roles of a user :/